src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogApiClient.java

```diff
@@ -319,14 +319,14 @@ private static final class ApiLogWriteStrategy implements LogWriteStrategy {
 
         public ApiLogWriteStrategy(String logIntakeUrl, Secret apiKey, HttpClient httpClient) {
             Map<String, String> headers = Map.of(
-                    "DD-API-KEY", Secret.toString(apiKey),
-                    "Content-Encoding", "gzip");
-            JsonPayloadSender<JSONObject> payloadSender = new CompressedBatchSender<>(
+                    "DD-API-KEY", Secret.toString(apiKey));
+            JsonPayloadSender<JSONObject> payloadSender = new BatchSender<>(
                     httpClient,
                     logIntakeUrl,
                     headers,
                     PAYLOAD_SIZE_LIMIT,
-                    Function.identity());
+                    Function.identity(),
+                    true);
```

Comment:
Could we create a constant for this `true` variable to improve readability? Like `private static final boolean compressionEnabled` or something similar.

Reply:
Created a named constant